locust/contrib/geventhttpclient.py

```diff
@@ -62,7 +75,7 @@ def request(self, method, path, name=None, **kwargs):
         
         try:
             response = self.client.urlopen(url, method=method, **kwargs)
-        except (ConnectionError, ConnectionRefusedError) as e:
+        except (ConnectionError, ConnectionRefusedError, socket.error) as e:
             # record the consumed time
             request_meta["response_time"] = int((time() - request_meta["start_time"]) * 1000)
```

Comment:
you can use `timeit.default_timer()` instead of `time.time()` to get the most accurate timer on the current platform (time.time() is not accurate on Windows, and better timers were introduced in Python 3)

Reply:
Nice, fixed!